app/src/main/java/com/example/palletify/ui/generator/GeneratorViewModel.kt

```diff
@@ -85,9 +88,8 @@ class GeneratorViewModel : ViewModel() {
             mode
         );
 
-        // For now, reset the mode to ANY, change this once we have a picker
-        mode = GenerationMode.ANY;
```

Comment:
i think we should keep it such that if the mode starts off as "ANY" then we want to set it back to "ANY" so that it displays in the dropdown

Reply:
the original comment was more intended for this and as a placeholder since ANY means a random colour generation method, and the GenerationMode.RANDOM means "truly random colours" that are put together

Reply:
Ahh got it. I'll make this change.